src/gui/src/mainWindow.cpp

```diff
@@ -744,26 +767,6 @@ void MainWindow::fit()
   fit_->trigger();
 }
 
-Selected MainWindow::makeSelected(std::any object, void* additional_data)
```

Comment:
What was the motivation to move this?

Reply:
When moving the `descriptors_` to the GUI class it was easier to move this along with it.